extern/test_bmi_fortran/src/bmi.f90

```diff
@@ -0,0 +1,565 @@
+! The Basic Model Interface (BMI) Fortran specification.
```

Comment:
This looks like a file used durring debugging as almost all of the bmi interface is disabled. Should this be in the pull request and if so is it up to date?

Reply:
It doesn't have to be for the purpose of the test model and the use of the ISO C library.  But to build the test model "stand alone" it would be required.  I forgot to uncomment the interface when I was debugging the module issue.  Ideally, the BMI dependency would be managed entirely out of source, but in cases where a model "customizes" the implementation of BMI by ignoring some functions (via commenting out the code) then it is required to carry around the definition.  I pushed a change to uncommment the relevant interface functions to make it useful, but I could easily be convinced to drop it from the PR.  Any thoughts?